crates/bevy_ui/Cargo.toml

```diff
@@ -29,7 +29,8 @@ bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
 
 # other
 stretch = "0.3.2"
-serde = {version = "1", features = ["derive"]}
+serde = { version = "1", features = ["derive"] }
 smallvec = { version = "1.6", features = ["union", "const_generics"] }
 bytemuck = { version = "1.5", features = ["derive"] }
-crevice = { path = "../crevice", version = "0.8.0", features = ["glam"] }
\ No newline at end of file
+crevice = { path = "../crevice", version = "0.8.0", features = ["glam"] }
+wgpu = { version = "0.12.0" }
```

Comment:
same re version specification and the missing trailing newline nit.

Reply:
fixed